app/javascript/packages/address-search/index.tsx

```diff
@@ -7,6 +7,9 @@ import SpinnerButton, { SpinnerButtonRefHandle } from '@18f/identity-spinner-but
 import type { RegisterFieldCallback } from '@18f/identity-form-steps';
 import useSWR from 'swr/immutable';
 import { useDidUpdateEffect } from '@18f/identity-react-hooks';
+import { InPersonLocations } from '@18f/identity-document-capture';
+
+export { InPersonLocations } from '@18f/identity-document-capture';
```

Comment:
I think that `InPersonLocations` is very logically related to the `AddressSearch` component, to the point where I rather expect them both to live in the `identity-address-search` component. And then have `identity-document-capture` import `InPersonLocations` rather than the other way around. Thoughts?